price_feeds_poller.py

```diff
@@ -98,18 +99,25 @@ def handle_requestUpdate(
       print(f"   xx Transaction rejected: {ex}")
       return [ 0 ]
 
+    #print(f" > Tx. receipt   :", receipt)
+    
     # Check if transaction was succesful
     if receipt['status'] == False:
       print(f"   $$ Transaction reverted !!")
       return [ -1, tx.hex() ]
-    else:      
-      logs = contract.events.PriceFeeding().processReceipt(receipt, errors=DISCARD)
+    else:
       requestId = 0
-      if len(logs) > 0:
+      if len(receipt['logs']) > 0:
+        logs = contract.events.PriceFeeding().processReceipt(receipt, errors=DISCARD)
```

Comment:
```suggestion
      logs = contract.events.PriceFeeding().processReceipt(receipt, errors=DISCARD)
      if len(logs) > 0:
```

I think before this change this was slightly easier to understand, unless there are some problems when calling `processReceipt` with an empty `receipt['logs']`?